### DISCUSSION TITLE
reTerminal and Pi camera modules | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#658] [Q&A]
# reTerminal and Pi camera modules | Seeed Studio Wiki

reTerminal and Pi camera modules intergration

https://wiki.seeedstudio.com/reTerminal-piCam/

**Comment by bluewhitep** [comment]
Can't no working with global shutter camera

**Reply by bluewhitep** [reply]
I fixed it.
This method need working on Raspberry pi os Legacy
